concepts/55 Vue Components/20 Component Configuration Syntax/75 Vue Form Input Bindings Support.md

```diff
@@ -20,4 +20,15 @@ DevExtreme editors support the <a href="https://v3.vuejs.org/guide/forms.html" t
     }
     </script>
 
+    <!-- tab: App.vue (Composition API) -->
+    <template>
+        <DxTextBox v-model="bindingProperty" />
+    </template>
+
+    <script setup>
+    import DxTextBox from 'devextreme-vue/text-box';
+
+    let bindingProperty = 'Some value';
+    </script>
+
 Alternatively, you can [use the `sync` modifier](/concepts/55%20Vue%20Components/20%20Component%20Configuration%20Syntax/33%20Two-Way%20Property%20Binding.md '/Documentation/Guide/Vue_Components/Component_Configuration_Syntax/#Two-Way_Property_Binding').
```

Comment:
This part looks obsolete as there is no example with the `sync` modifier in the referenced topic